model/certificate.go

```diff
@@ -66,11 +66,9 @@ type Certificate struct {
 	ACMEKey          *ecdsa.PrivateKey
 }
 
-var ErrInvalidDomains = errors.New("invalid domains")
-var ErrInvalidEmail = errors.New("invalid email")
```

Comment:
Wait so are these errors gone? Aren't they being referenced somewhere else?

Reply:
They were just moved upwards in the file.